addons/im_livechat/static/src/embed/common/store_service_patch.js

```diff
@@ -30,3 +30,6 @@ patch(Store.prototype, {
         return params;
     },
 });
+
+pyToJsModels["chatbot.script.step"] = "ChatbotScriptStep";
+pyToJsModels["chatbot.script.answer"] = "ChatbotScriptStepAnswer";
```

Comment:
`pyToJsModels` is deprecated now that `_name` exists on models, we should not introduce new one